src/content/en/fundamentals/media/mobile-web-video-playback.md

```diff
@@ -0,0 +1,611 @@
+is hidden. This happens when screen lock is active or when you switch tabs for
+instance.
+
+    document.addEventListener('visibilitychange', function() {
+      // Pause video when page is hidden.
+      if (document.hidden) {
+        video.pause();
+      }
+    });
+
+Note: Chrome for Android already pauses videos when page is hidden.
+
+### Show/hide mute button on video visibility change
+
+If you use the new [Intersection Observer API], you can be even more granular
+at no cost. This API lets you know when an observed element enters or exits the
+browser's viewport.
+
+Let's show/hide a mute button based on the video visibility in the page. If
+video is playing but not currently visible, a mini mute button will be shown in
+the bottom right corner of the page to give user control over video sound. The
+`volumechange` video event is used to update the mute button styling.
+
+    <button id="muteButton"></button>
+
+<div class="clearfix"></div>
+
+    if ('IntersectionObserver' in window) {
+      // Show/hide mute button based on video visibility in the page.
+      function onIntersection(entries) {
+        entries.forEach(function(entry) {
+          muteButton.hidden = video.paused || entry.isIntersecting;
+        });
+      }
+      var observer = new IntersectionObserver(onIntersection);
+      observer.observe(video);
+    }
+
+    muteButton.addEventListener('click', function() {
+      // Mute/unmute video on button click.
+      video.muted = !video.muted;
+    });
+
+    video.addEventListener('volumechange', function() {
+      muteButton.classList.toggle('active', video.muted);
+    });
+
+<video controls controlsList="nodownload" muted playsinline>
+  <source src="/web/fundamentals/media/videos/video-visibility.webm"
+          type="video/webm">
+  <source src="/web/fundamentals/media/videos/video-visibility.mp4"
+          type="video/mp4">
+</video>
+
+### Customize Media Notifications
+
+With the [Media Session API]{: .external}, you can also customize media
+notifications by providing metadata for the currently playing video. It also
+allows you to handle media related events such as seeking or track changing
+which may come from notifications or media keys. To see this in action, check
+out the [sample]{: .external }.
+
+When your web app is playing audio or video, you can already see a media
+notification sitting in the notification tray. On Android, Chrome does its best
+to show appropriate information by using the document's title and the largest
+icon image it can find.
+
+Let's see how to customize this media notification by setting some media
+session metadata such as the title, artist, album name, and artwork with the
+[Media Session API]{: .external }.
+
+<pre class="prettyprint">
+playPauseButton.addEventListener('click', function() {
+  event.stopPropagation();
+  if (video.paused) {
+    video.play()
+    <strong>.then(function() {
+      setMediaSession();
+    });</strong>
```

Comment:
nit: this could be written as `video.play().then(setMediaSession)`, right? I bet I must be missing some subtleties :)